app/helpers/application_helper.rb

```diff
@@ -1,2 +1,12 @@
 module ApplicationHelper
+
+  def flash_class(level)
+    case level
+        when :notice then "alert alert-info"
```

Comment:
I like the use of the `case` here 👍  

You'll want to amend to two space tabbing:

```ruby
def flash_class(level)
  case level
    when :notice then "alert alert-info"
    when :success then "alert alert-success"
    when :error then "alert alert-error"
    when :alert then "alert alert-error"
  end
end
```

Alternatively, you'll often see this formatting:

```ruby
def flash_class(level)
  case level
  when :notice 
    "alert alert-info"
  when :success 
    "alert alert-success"
  when :error 
    "alert alert-error"
  when :alert 
    "alert alert-error"
  end
end
```

Your call.